pkg/controller/nodelifecycle/metrics.go

```diff
@@ -60,11 +61,21 @@ var (
 		[]string{"zone"},
 	)
 	evictionsNumber = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem:         nodeControllerSubsystem,
+			Name:              evictionsNumberKey,
+			Help:              "Number of Node evictions that happened since current instance of NodeController started. This metric is replaced by node_collector_evictions_total",
+			DeprecatedVersion: "1.23.0",
+			StabilityLevel:    metrics.ALPHA,
+		},
+		[]string{"zone"},
+	)
+	evictionsTotal = metrics.NewCounterVec(
 		&metrics.CounterOpts{
 			Subsystem:      nodeControllerSubsystem,
-			Name:           evictionsNumberKey,
+			Name:           evictionsTotalKey,
 			Help:           "Number of Node evictions that happened since current instance of NodeController started.",
-			StabilityLevel: metrics.ALPHA,
+			StabilityLevel: metrics.STABLE,
```

Comment:
why we make this new metrics as stable?

Reply:
Please refer to: https://github.com/kubernetes/kubernetes/pull/105935#issuecomment-952951988.